minecraft_servers/veranynetwork/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "server_name": "veranynetwork",
+  "nice_name": "VeranyNET",
+  "direct_ip": "dedi.verany.net",
+  "server_wildcards": [
+    "%.verany.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://eynoah.de/imprint",

```

Comment:
Not a valid website. Please use a website that presents the server or delete the line.